### PR TITLE
Bump readme for 0.7

### LINKED ISSUE #539: Update README for 0.7 release
The README needs some updates for the 0.7 release:
- Add links to new font renderer crates (if they are finished for 0.7)
- Update instructions how to run simulator examples (#536)
- Change font attribution
- Link to new migration guide

Please add any additional items that I forgot.

### PR DESCRIPTION
**Pull request by jamwaffles**
Thank you for helping out with embedded-graphics development! Please:

- [x] Check that you've added passing tests and documentation
- [x] Add a `CHANGELOG.md` entry in the **Unreleased** section under the appropriate heading (**Added**, **Fixed**, etc) if your changes affect the **public API**
- [x] Run `rustfmt` on the project
- [x] Run `just build` (Linux/macOS only) and make sure it passes. If you use Windows, check that CI passes once you've opened the PR.

## PR description

Update the readme as per #539. I'm not going to shuffle the sections around as the last comment(s) mention as it's not critical for release, and they can be moved later in a patch release.

Closes #539.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -138,29 +142,34 @@
...
+//! #   display.set_allow_overdraw(true);
 //!
 //!     // Create styles used by the drawing operations.
 //!     let thin_stroke = PrimitiveStyle::with_stroke(BinaryColor::On, 1);
 //!     let thick_stroke = PrimitiveStyle::with_stroke(BinaryColor::On, 3);
+//!     let border_stroke = PrimitiveStyleBuilder::new()
+//!         .stroke_width(3)
+//!         .stroke_alignment(StrokeAlignment::Inside)
+//!         .stroke_color(BinaryColor::On)
+//!         .build();
```
Comment (rfuest) [inline src/lib.rs:162]: I like to keep the calls in the same order as the parameters in the `with_stroke` above, which makes it easier to spot the similarity between the statements IMO:

```suggestion
//!     let border_stroke = PrimitiveStyleBuilder::new()
//!         .stroke_color(BinaryColor::On)
//!         .stroke_width(3)
//!         .stroke_alignment(StrokeAlignment::Inside)
//!         .build();
```

File: src/lib.rs [hunk 2]
```diff
@@ -113,6 +113,10 @@
 //!
 //! Please read [the migration guide](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
 //!
+//! # Migrating from 0.6 to 0.7
+//!
+//! Please read [the migration guide](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.6-0.7.md).
+//!
```
Comment (rfuest) [inline src/lib.rs:119]: We could merge both migration guide links into one section.

```suggestion
//! # Migrating from older version
//!
//! * [Migration guide from 0.5 to 0.6](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
//! * [Migration guide from 0.6 to 0.7](https://github.com/embedded-graphics/embedded-graphics/blob/master/MIGRATING-0.5-0.6.md).
//!
```

**Review by rfuest — CHANGES_REQUESTED** [review]
Looks good overall, except for a few nits and:

* The example doesn't build correctly
* The font attribution section needs to be updated
* The examples are no longer in the `simulator`  repo

**Comment by jamwaffles** [comment]
I should've tested the build locally :facepalm: I knew there was a reason I bought a 5950X...

I've applied the suggestions too - thanks for the review.

**Review by rfuest — CHANGES_REQUESTED** [review]
> I should've tested the build locally facepalm I knew there was a reason I bought a 5950X...

Nice, I thought you wanted to buy a 5900X. Did you upgrade because the 5900X wasn't available?

The "Simulator" section still contains instructions how to use the examples in the simulator repo.

Should we remove the "Attribution" section entirely or update it to link to the xorg repo (https://gitlab.freedesktop.org/xorg/font/misc-misc/)?

**Comment by jamwaffles** [comment]
> Did you upgrade because the 5900X wasn't available?

Precisely this. My credit card isn't too happy but I'm glad I could get _something_...

I changed the simulator section to point to the examples repo, and removed the attribution section entirely.

I also updated the screenshot, but now that I put the commit hash in the URL, I think this will break immediately after merge, so I'll open another PR to update the hash when we're done here.

I gotta go do a BBQ so I'll be AFK for a while after this review pass.

**Comment by rfuest** [comment]
> Precisely this. My credit card isn't too happy but I'm glad I could get _something_...

I had considered to do the same, but I'm still stuck with my ancient CPU. Without being able to buy a matching graphics card for a reasonable price in the foreseeable future I couldn't bring myself to spend the extra money.

**Review by rfuest — APPROVED** [review]
LGTM

**Comment by jamwaffles** [comment]
Danke!

**Pull request merged**